editor/resources/liveupdate-meta.properties

```diff
@@ -1,86 +1,65 @@
+[]
+localization-prefix = liveupdate
+
 [liveupdate]
-title = Live Update
 
 mode.type = string
 mode.default = Amazon
-mode.help = Target
 mode.options = Zip, Amazon, Folder
 
 amazon-credential-profile.type = string
-amazon-credential-profile.help = Amazon credentials profile
 amazon-credential-profile.default = 
-amazon-credential-profile.presentation-category = Amazon
+amazon-credential-profile.presentation-category = amazon
 
 amazon-bucket.type = string
-amazon-bucket.help = Amazon Bucket
 amazon-bucket.default =
-amazon-bucket.presentation-category = Amazon
+amazon-bucket.presentation-category = amazon
 
 amazon-prefix.type = string
-amazon-prefix.help = Amazon Prefix
 amazon-prefix.default =
-amazon-prefix.presentation-category = Amazon
+amazon-prefix.presentation-category = amazon
 
 save-zip-in-bundle-folder.type = boolean
-save-zip-in-bundle-folder.title = Save zip in bundle folder
-save-zip-in-bundle-folder.help = Moves zip into bundle directory after the bundle process
```

Comment:
I removed `.help`, `.title`, and `.label` keys from here, but not from meta.properties, because those can still be used in the command line bob (unlocalized), but these properties are editor only, and it's better not to duplicate strings.